scripts/api-compatibility-check/README.md

```diff
@@ -0,0 +1,83 @@
+# API Compatibility check
```

Comment:
```suggestion
# API Compatibility Check
```

We should be more consistent with capitalization in the headings. I'm fine with sentence case or title case as long as it's consistent.